src/main/java/db/Storage.java

```diff
@@ -0,0 +1,27 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitList = new HashMap<>();
+
+    public static void add(String fruit, Integer amount) {
+        fruitList.put(fruit, fruitList.getOrDefault(fruit, 0) + amount);
+    }
+
+    public static void remove(String fruit, Integer amount) {
+        if (!fruitList.containsKey(fruit) || fruitList.get(fruit) < amount) {
+            throw new RuntimeException("There's not enough " + fruit + " in storage.");
```

Comment:
Let's use more appropriate exceptions
```suggestion
            throw new IllegalArgumentException("There's not enough " + fruit + " in storage.");
```